templates/traas.yaml

```diff
@@ -95,15 +100,40 @@ resources:
       type: string
       value: |
         #!/bin/bash
+        # Ensure dev requirements for pre-provisioned nodes
+        setenforce 0
+        sed -i 's/^SELINUX=.*/SELINUX=permissive/' /etc/selinux/config
```

Comment:
this is not needed in traas and ought to be getting set via tripleo-ci. Is that not the case?